src/modules/organizations/containers/organization-container.jsx

```diff
@@ -22,13 +23,25 @@ class OrganizationContainer extends React.Component {
     this.props.dispatch(setCurrentOrganization(null));
   }
 
+  updateOrganization(organizationFields) {
+    organizationFields.name = organizationFields.display_name; // eslint-disable-line no-param-reassign
+    organizationFields.title = organizationFields.name.replace(/\W/g, ''); // eslint-disable-line no-param-reassign
```

Comment:
Ok, I think we need to understand how the API handles this field since part of my confusion had to do with the fact that PFE isn't using it with projects at all. If you fetch a project, the title field is returned, but is identical to display_name, so something is setting that field, but not the front end.